azurerm/resource_arm_kubernetes_cluster_test.go

```diff
@@ -795,6 +832,69 @@ resource "azurerm_kubernetes_cluster" "test" {
 `, rInt, location, rInt, rInt, rInt, clientId, clientSecret)
 }
 
+func testAccAzureRMKubernetesCluster_windowsProfile(rInt int, clientId string, clientSecret string, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_kubernetes_cluster" "test" {
+  name                = "acctestaks%d"
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  dns_prefix          = "acctestaks%d"
+  kubernetes_version  = "1.14.0"
+
+  linux_profile {
+    admin_username = "acctestuser%d"
+
+    ssh_key {
+      key_data = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCqaZoyiz1qbdOQ8xEf6uEu1cCwYowo5FHtsBhqLoDnnp7KUTEBN+L2NxRIfQ781rxV6Iq5jSav6b2Q8z5KiseOlvKA/RF2wqU0UPYqQviQhLmW6THTpmrv/YkUCuzxDpsH7DUDhZcwySLKVVe0Qm3+5N2Ta6UYH3lsDf9R9wTP2K/+vAnflKebuypNlmocIvakFWoZda18FOmsOoIVXQ8HWFNCuw9ZCunMSN62QGamCe3dL5cXlkgHYv7ekJE15IA9aOJcM7e90oeTqo+7HTcWfdu0qQqPWY5ujyMw/llas8tsXY85LFqRnr3gJ02bAscjc477+X+j/gkpFoN1QEmt terraform@demo.tld"
+    }
+  }
+
+  windows_profile {
+    admin_username = "azureuser"
+    admin_password = "pass_123-worD"
+  }
+
+  agent_pool_profile {
+    name    = "linux"
+	type            = "VirtualMachineScaleSets"
+    count   = "1"
+    vm_size = "Standard_DS2_v2"
+	max_pods = 30
+    os_type = "Linux"
+	os_disk_size_gb = "30"
+  }
+
+  agent_pool_profile {
+    name    = "win"
+	type            = "VirtualMachineScaleSets"
```

Comment:
can Windows nodes be powered by AvailabilitySet too?

Reply:
No, this is a limitation of Windows nodes. Only supported with VMSS

Reply:
👍 could we document this in the documentation e.g.

```
-> **NOTE:** The Windows Agent Pool Profile only supports a `type` of `VirtualMachineScaleSets`
```

Reply:
Could we add this to the documentation as suggested?